test/narrative_util_test.py

```diff
@@ -16,267 +15,251 @@
 )
 from StaticNarrative.narrative_ref import NarrativeRef
 
+USER_ID = "some_user"
+TOKEN = "some_token"  # noqa: S105
+REF = NarrativeRef.parse("1/2/3")
 
-class NarrativeUtilTestCase(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        cls.user_id = "some_user"
-        cls.token = "some_token"
-        cls.cfg = get_test_config()
-
-    @requests_mock.Mocker()
-    def test_read_narrative_ok(self, rqm):
-        ref = "43666/1/18"
-        ref_to_file = {ref: "data/43666/narrative-43666.1.18.json"}
-        set_up_ok_mocks(rqm, ref_to_file=ref_to_file)
-        nar = read_narrative(
-            NarrativeRef.parse("43666/1/18"),
-            Workspace(url=self.cfg["workspace-url"], token=self.token),
+
+def test_read_narrative_ok(workspace_url: str, requests_mock) -> None:
```

Comment:
no real changes here; switched over to `with pytest.raises(err_type, match="match string"):` for the failure tests so no need for the `assert ... in str(e.exception)`